rotkehlchen/serialization/deserialize.py

```diff
@@ -578,10 +607,36 @@ def deserialize_evm_transaction(
             gas_used = read_integer(data, 'gasUsed', source)
         nonce = read_integer(data, 'nonce', source)
 
-        if chain_id in L2_CHAINIDS_WITH_L1_FEES and evm_inquirer is not None:
-            if not raw_receipt_data:
-                raw_receipt_data = evm_inquirer.get_transaction_receipt(tx_hash)
-            l1_fee = maybe_read_integer(raw_receipt_data, 'l1Fee', source)
+        if chain_id in L2_CHAINIDS_WITH_L1_FEES:
+            if (
+                (l1_fee := data.get('L1FeesPaid')) is None and  # etherscan txlist uses this key
+                evm_inquirer is not None
+            ):
+                if raw_receipt_data is None:
+                    raw_receipt_data = _get_transaction_receipt(
+                        tx_hash=tx_hash,
+                        chain_id=chain_id,
+                        timestamp=timestamp,
+                        evm_inquirer=evm_inquirer,
+                    )
```

Comment:
I don't get why you need to do this again. Isn't it the same as what you do in line 594? You call the same logic and it would use the official node so it should have it. If it doesn't you just need to query etherscan. I don't see how this is needed again